arithmetization/src/main/java/net/consensys/linea/sequencer/txselection/selectors/MaxBlockCallDataTransactionSelector.java

```diff
@@ -53,11 +53,13 @@ public TransactionSelectionResult evaluateTransactionPreProcessing(
     final int transactionCallDataSize = transaction.getPayload().size();
 
     if (isTransactionExceedingBlockCallDataSizeLimit(transactionCallDataSize)) {
-      log.trace(
-          "BlockCallData {} greater than {}, completing operation",
-          transactionCallDataSize,
-          maxBlockCallDataSize);
-      return BLOCK_CALLDATA_FULL;
+      log.atTrace()
+          .setMessage(
+              "BlockCallData including this tx is {} greater than the max allowed {}, skipping tx")
+          .addArgument(() -> blockCallDataSize + transactionCallDataSize)
```

Comment:
Is here blockCallDataSize or  blockCallDataSize + transactionCallDataSize ?

Reply:
the latter, since `blockCallDataSize` still does not contain the current tx calldata size

Reply:
I believe it is already added with onTransactionSelected, right ?

Reply:
no, the sum is done at last step when the tx is selected in the `onTransactionSelected`